app/helpers/template_helpers.py

```diff
@@ -79,7 +79,7 @@ def render_template(template, **kwargs):
         languages=get_languages_context(),
         schema_theme=theme,
         language_code=get_locale().language,
-        survey_title=cookie_session.get("survey_title"),
+        survey_title=lazy_gettext("Census 2021"),
```

Comment:
- `cookie_session.get("survey_title")` is called earlier in this file to set the page header context title - that also needs to be changed.
- Setting of `survey_title` in the cookie needs to be removed from `session.py`.

Reply:
I  set a new variable _title_ to `lazy_gettext("Census 2021")` to avoid repetition. Session's _survey_title_ removed and cookie test changed, thanks!👍